virtcontainers/kata_agent.go

```diff
@@ -1019,6 +1019,12 @@ func constraintGRPCSpec(grpcSpec *grpc.Spec, passSeccomp bool) {
 		grpcSpec.Linux.Seccomp = nil
 	}
 
+	// Disable selinux
+	if grpcSpec.Process.SelinuxLabel != "" {
+		k.Logger().Warnf("Selinux label specified in config, but not supported in Kata yet, running container without selinux")
```

Comment:
nitpick, for which I don't know if you might fail the CI - but, you don't need `Warnf` if you have no % formatters in the string - you might be able to just use `Warn` :-)

Reply:
Looks good to me with this change, but I am not a member, so I cannot approve.